plugins/GCodeWriter/GCodeWriter.py

```diff
@@ -68,23 +68,21 @@ def _serialiseSettings(self, settings):
         prefix = ";SETTING_" + str(GCodeWriter.version) + " "  # The prefix to put before each line.
         prefix_length = len(prefix)
 
-        all_settings = InstanceContainer("G-code-imported-profile") #Create a new 'profile' with ALL settings so that the slice can be precisely reproduced.
-        all_settings.setDefinition(settings.getBottom())
-        for key in settings.getAllKeys():
-            all_settings.setProperty(key, "value", settings.getProperty(key, "value")) #Just copy everything over to the setting instance.
-        serialised = all_settings.serialize()
+        global_stack = Application.getInstance().getGlobalContainerStack()
+        container_with_profile = global_stack.findContainer({"type": "quality"})
+        serialized = container_with_profile.serialize()
```

Comment:
I know. Just followed Arjen's suggestion to save ’saved’ profiles first.
We can add an additional ticket for this on JIRA, so they (Arjen, Tim and
Jasper)  can decided when we want this fully supported.